shared/src/business/useCases/practitioners/getPractitionerDocumentDownloadUrlInteractor.test.ts

```diff
@@ -0,0 +1,64 @@
+import { ROLES } from '../../entities/EntityConstants';
+import { UnauthorizedError } from '../../../errors/errors';
+import { applicationContext } from '../../test/createTestApplicationContext';
+import { getPractitionerDocumentDownloadUrlInteractor } from './getPractitionerDocumentDownloadUrlInteractor';
+
+describe('getPractitionerDocumentDownloadUrlInteractor', () => {
+  let testUser;
+  const mockDocumentMetadata = {
+    categoryName: 'Application',
+    categoryType: 'Application',
+    description: 'hubba bubba bubble gum',
+    fileName: 'application.pdf',
+    location: undefined,
+    practitionerDocumentFileId: '07044afe-641b-4d75-a84f-0698870b7650',
+  } as any;
+
+  const mockPractitioner = { barNumber: 'PT1234' };
+
+  beforeAll(() => {
+    applicationContext
+      .getPersistenceGateway()
+      .getPractitionerDocumentByFileId.mockReturnValue({
+        fileName: 'SomeFile.pdf',
+      });
+  });
+
+  it('should throw an unauthorized error when the user does not have permission to download the practitioner documentation file', async () => {
+    testUser = {
+      role: ROLES.petitioner,
+      userId: 'petitioner',
+    };
+
+    applicationContext.getCurrentUser.mockImplementation(() => testUser);
+
+    await expect(
+      getPractitionerDocumentDownloadUrlInteractor(applicationContext, {
+        barNumber: mockPractitioner.barNumber,
+        practitionerDocumentFileId:
+          mockDocumentMetadata.practitionerDocumentFileId,
+      }),
+    ).rejects.toThrow(UnauthorizedError);
+  });
+
+  it('should not throw an unauthorized error when the user does not have permission to download the practitioner documentation file', async () => {
```

Comment:
can you rephrase this `it` statement? I think it's meant to be:

> should not throw an unauthorized error when the user does has permission to download the practitioner documentation file

or perhaps better

> successfully download the document when the user has permission to download the practitioner documentation file

Reply:
rephrased to 'should not throw an unauthorized error when the user has permission to download the practitioner documentation file'